checker/tests/nullness/java8inference/Issue402.java

```diff
@@ -3,13 +3,14 @@
 
 // @below-java8-jdk-skip-test
 
-// This test now seems to fail because of Issue 1006
-// @skip-test
-
 import java.util.Comparator;
 import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
 
+@SuppressWarnings({
+    "nullness",
+    "keyfor"
+}) // Type argument inference does not infer the correct types.
```

Comment:
I would add the explanation before the SuppressWarning.
Can we add an issue number for when we expect this to be fixed. Is it #979?